docs/Github-workflow.md

```diff
@@ -0,0 +1,61 @@
+[Back to overview](https://github.com/WordPress/performance/blob/trunk/docs/README.md)
```

Comment:
Nit-pick, but it's better to have this a relative URL so that it works no matter which branch you're looking at.
```suggestion
[Back to overview](./README.md)
```